start-vms

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir -p inventories
+vagrant up $@
+
+./vagrant.py --yaml > inventories/local_vagrant
```

Comment:
`start-vms` script currently overwrites `inventories/local_vagrant` without any checks, should we preserve the existing inventory file when regenerating?

Reply:
Since the intent is that this script controls the creation of these VMs, I do not see a reason to preserve the inventory in any way.